pandas/tests/arrays/masked/test_arithmetic.py

```diff
@@ -52,6 +54,11 @@ def test_array_scalar_like_equivalence(data, all_arithmetic_operators):
 def test_array_NA(data, all_arithmetic_operators):
     if "truediv" in all_arithmetic_operators:
         pytest.skip("division with pd.NA raises")
+    if (
+        "floordiv" in all_arithmetic_operators
+        or "rfloordiv" in all_arithmetic_operators
```

Comment:
since this is already true if "floordiv" in all_arithmetic_operators since `"floordiv" in "rfloordiv"` is `True`

Reply:
Thats a good point, would go with this then?